applications/Unity.GrantManager/src/Unity.GrantManager.Web/appsettings.Development.json

```diff
@@ -13,8 +13,18 @@
     "IsJobExecutionEnabled": false,
     "Quartz": {
       "IsAutoRegisterEnabled": false
+    },
+    "IntakeResync": {      
+      "Expression": "0 0/1 * * * ?"
     }
   },
+  "Notifications": {
+    "TeamsNotificationsWebhook": "",
+    "ChesUrl": "https://ches-dev.api.gov.bc.ca/api/v1",
+    "ChesTokenUrl": "https://dev.loginproxy.gov.bc.ca/auth/realms/comsvcauth/protocol/openid-connect/token",
+    "ChesClientId": "",
+    "ChesClientSecret": ""
+  },
   "Serilog": {
     "MinimumLevel": {
       "Default": "Debug",

```

Comment:
[21:42:35 DBG] ({SourceContext="Quartz.Core.QuartzSchedulerThread"}) Batch acquisition of 0 triggers
[21:42:35 DBG] ({SourceContext="Quartz.Core.JobRunShell"}) Calling Execute on job DEFAULT.Volo.Abp.BackgroundJobs.BackgroundJobWorker
Lots of these
![image](https://github.com/bcgov/Unity/assets/39963238/f4459770-90da-4199-a63a-cf8e418ce2b5)